### PR TITLE
CCOM-1: Create Pull Request Template

### PR DESCRIPTION
**Pull request by adamkroonbcgov**
![TICKETS_AND_ISSUES]

[[JIRA_BOARD_ABBREVIATION]-[JIRA_TASK_NUMBER]: [TITLE_OF_JIRA_TASK]](https://link-to-jira-board-task)

![DESCRIPTION]

Plain language explanation of what feature request, bug fix, or enhancement this pull request includes

![CHECKLIST]
- [ ] By checking this, I can confirm that I have done the following tasks before opening this pull request:

- 📚 My Pull Request Title follows this pattern: 
    ```[FEATURE_REQUEST|ENHANCEMENT|BUG_FIX|DOCS] [JIRA_BOARD_NUMBER]: [TITLE_OF_JIRA_TASK]```

    Example:
    ```FEATURE_REQUEST MVP1-1: Change email address```
- 🔎 I have self-reviewed my code.
- 📚 I have updated the relevant documentation.
- 🧪 My code passes unit tests that are present in this repository.
- 💅 My code passes standardized linting checks.
- 👥 I am complying with our [Code of Conduct](REPLACE_WITH_URL_TO_PROJECT_CODE_OF_CONDUCT).

[DESCRIPTION]: https://img.shields.io/badge/description-A23006?style=for-the-badge
[TICKETS_AND_ISSUES]: https://img.shields.io/badge/tickets_&_issues-0859A1?style=for-the-badge
[CHECKLIST]: https://img.shields.io/badge/checklist-AD0031?style=for-the-badge

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+![DESCRIPTION]
+
+Plain language explanation of what feature request, bug fix, or enhancement this pull request includes
+
+![CHECKLIST]
+- [ ] By checking this, I can confirm that I have done the following tasks before opening this pull request:
+
+- 📚 My Pull Request Title follows this pattern: 
+    ```[FEATURE_REQUEST|ENHANCEMENT|BUG_FIX|DOCS] [JIRA_BOARD_NUMBER]: [TITLE_OF_JIRA_TASK]```
```
Comment (mmsgov) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: An example of a title here would be good to have, to make sure we get the formatting right (I'm a bit of a stickler and don't like small variations in formatting). Something like FEATURE_REQUEST MVP1-1: Change email address

Comment (adamkroonbcgov) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: use the suggestion tool ^^ and I'll commit your change @milosdespotovic

Comment (mmsgov) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: I hope this formatting works out properly! 

```suggestion
    ```[FEATURE_REQUEST|ENHANCEMENT|BUG_FIX|DOCS] [JIRA_BOARD_NUMBER]: [TITLE_OF_JIRA_TASK]```
    Example:
    ```FEATURE_REQUEST MVP1-1: Change email address```
```

**Comment by dbarkowsky** [comment]
The checklist area feels a little cluttered. I totally missed the checkbox on my first pass amongst the emojis.

**Comment by Bottle7** [comment]
The emojis are nice and putting them at the end of the text might alleviate the issue of missing the checkbox.

We should probably have standardized abbreviations for the feature | enhancement | bug etc. so that we can stay within the character limit on the title.

**Pull request closed without merging**